src/reflow/mod.rs

```diff
@@ -252,16 +252,16 @@ r#" This module contains documentation thats
 
     #[test]
     fn reflow_markdown() {
```

Comment:
make this a `macro_rules`, we will have more of those :)

Reply:
What do you mean by trailing double spaces on soft brakes?

I'll start with adding tests for all of them..

Reply:
regarding links: If the line brake would occur in the link text, markdown can handle it (at least the github-flavoured markdown: https://gist.github.com/KuabeM/9f71b5ba97d90b0645d77167c5387148), but the link itself must be kept intact.